tools/flake8-torch/flake8_torch/checker.py

```diff
@@ -0,0 +1,96 @@
+from pathlib import Path
+import ast
+import yaml
+
+
+class TorchVisitor(ast.NodeVisitor):
+    def __init__(self):
+        # Map from import alias (may be the same as name) to import name
+        self._import_info = {}
+        self._call_info = []
+
+    def visit_Call(self, node):
+        func_name = ""
+        curr = node.func
+        while isinstance(curr, ast.Attribute):
+            func_name = "." + curr.attr + func_name
+            curr = curr.value
+        if isinstance(curr, ast.Name):
+            func_name = curr.id + func_name
+
+        self._call_info.append(
+            {"name": func_name, "lineno": node.lineno, "col_offset": node.col_offset}
+        )
+
+        self.generic_visit(node)
+
+    def visit_Import(self, node):
+        for alias in node.names:
+            if alias.asname is None:
+                self._import_info[alias.name] = alias.name
+            else:
+                self._import_info[alias.asname] = alias.name
+
+        self.generic_visit(node)
+
+    def visit_ImportFrom(self, node):
+        module = "" if node.module is None else node.module
+        module = "." * node.level + module
+        for alias in node.names:
+            if alias.asname is None:
```

Comment:
Is `asname` guaranteed to be defined? If not, use `getattr`
```suggestion
            if getattr(alias, 'asname', None) is None:
```

Reply:
`asname` is always defined, but can be None: https://docs.python.org/3/library/ast.html#ast.alias